sopel/plugins/handlers.py

```diff
@@ -622,10 +624,14 @@ def get_version(self) -> Optional[str]:
         """
         version: Optional[str] = super().get_version()
 
+        # note: we need to check for attribute because of older Python version
+        # and we need to check if .dist is not None because hasattr does not
+        # type safeguard properly (or mypy doesn't care?)
         if (
             version is None
-            and hasattr(self.entry_point, "dist")
-            and hasattr(self.entry_point.dist, "name")
+            and hasattr(self.entry_point, 'dist')
+            and self.entry_point.dist is not None
+            and hasattr(self.entry_point.dist, 'name')
```

Comment:
This could have been a +1 diff instead of -2/+3 😂

Reply:
Will revert that.